src/api/api_auth_shibuqam.nit

```diff
@@ -94,7 +99,10 @@ class ShibCallback
 			config.players.save player
 		end
 		session.player = player
-		res.redirect "/player"
+
+		# Redirect
+		var page = session.auth_next or else "/player"
```

Comment:
AuthHadler is not documented, I not sure what is its role but it seems unrelated to the classes GithubLogin, GithubCallback, ShibLogin and ShibCallback.